client/src/main/java/com/vesoft/nebula/client/storage/processor/EdgeProcessor.java

```diff
@@ -39,7 +38,7 @@
                     Map<String, Object> props = Maps.newHashMap();
```

Comment:
why not directly to use the ValueWrapper for property value?

Reply:
Object contains data‘s type and value，property value no need to be wrapped.
If use ValueWrapper for property, then unbox is also needed to get the real value.

Reply:
ValueWrapper also contains data‘s type and value. the object also need to judge the type then to get the value, it's not different of them to get value, but if use ValueWrapper, the string is decode when the user gets, but the object is no. and ValueWrapper has processed all type of value, you no need do it again.

Reply:
The object returned to the user has been encoded, and the String type is not in the form of byte[]. When getting the value of the data, you can get the value of the Object directly. If use the valueWrapper,  need to call the method to get the value of the data.

Reply:
I'm not saying the object uses byte[], I'm saying you convert all the data whether the user uses it or not, not when the user gets it and then converts it. and the id use ValueWrapper, the prop value use object, the id and prop value is the same value. it is no necessary two data structures。